items/FAIRY_CHESTPLATE.json

```diff
@@ -14,11 +11,15 @@
     "",
     "§6Full Set Bonus: Fairy's Outfit",
     "§7Gain §c1❤ §7per Fairy Soul.",
-    "§7You found: §d16 Fairy Souls",
+    "§7You found: §d§k000§d Fairy Souls",
```

Comment:
does the obfuscated text actually get reset here?

Reply:
wdym by reset

Reply:
is the text after the numbers de-obfuscated?

Reply:
yes, it works as intended only the 000's get obfuscated